### ISSUE TITLE
How to speed up metadata updates

### ISSUE THREAD

**Issue by arnaldocan** [astrada/google-drive-ocamlfuse#686]
I'm using Google-drive-ocamlfuse version 0.7.23 and have made some tweaks to update up local cache, including: async_upload_queue, write_buffers max_cache_size_mb, max_memory_cache_size, metadata_cache_time, metadata_memory_cache and metadata_memory_cache_saving_interval.

In my setup, creating or updating a file is slow, for example:

```
$ time touch a.txt
real    0m1,997s
user    0m0,002s
sys     0m0,000s
```

```
$ time touch a.txt
real    0m0,808s
user    0m0,000s
sys     0m0,002s
```
However, updating already existing files is very fast:

```
$ time echo a > a.txt 
real    0m0,040s
user    0m0,000s
sys     0m0,000s
```

Is there a way to optimize metadata updating?

**Comment by astrada** [comment]
No, sorry, when you create a new file, the REST API is called, while when you update the content, the upload is queued. That's why the first case is slower than the second.

**Comment by arnaldocan** [comment]
Ok, thanks for the explanation.
